src/db/models.rs

```diff
@@ -144,3 +151,26 @@ impl Account {
                                                                 user=self.username).into())
     }
 }
+
+impl Status {
+    pub fn account(&self, db_conn: &Connection) -> QueryResult<Account> {
+        use db::schema::accounts::dsl;
+        dsl::accounts
+            .find(self.account_id)
+            .get_result::<Account>(&**db_conn)
```

Comment:
This is more a general diesel question, but how do you avoid n+1 queries when rendering results for lists of statuses?

Reply:
Not sure yet, tbh.

Reply:
You can do something like
```rs
let account = accounts.find(id).first::<Account>(&conn).unwrap();
let statuses = Status::belonging_to(&account).load::<Status>(&conn).unwrap();
```